S3-Credentials-Design.md

```
@@ -0,0 +1,367 @@
+                         session_token: Optional[str] = None):
+        pass
+    
+    def delete_credential(self, credential_id: int):
+        pass
+```
+
+### Credential manager script
+
+The `admin-tools/credential-manager.sh` script provides a command-line interface for managing AWS S3 credentials in the database. It follows the same pattern as other admin tools in the CLP package, acting as a wrapper around a Python script that performs the actual operations.
+
+**Command line arguments**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)